azure-pipelines-dependency-graph-testing.yml

```diff
@@ -245,13 +245,23 @@ stages:
 
           - script: |
               # Install dependencies
-              pip install pytest-cov pytest-icdiff pytest-mock pytest-azurepipelines invoke
+              pip install -r requirements-dev-lite.txt
 
               # Run unit-tests
               invoke tests --ci --cloud
 
             displayName: 'Unit Tests'
 
+          - script: |
+              pip install coveralls
+              export CIRCLE_BRANCH=$BUILD_SOURCEBRANCH
+              coveralls
+            displayName: 'coveralls'
+            env:
+              CIRCLECI: 1
+              CIRCLE_BUILD_NUM: $(Build.BuildNumber)
+              COVERALLS_REPO_TOKEN: $(coveralls_repo_token)
```

Comment:
We are mimicking circleci here
https://coveralls-python.readthedocs.io/en/latest/usage/configuration.html#azure-pipelines-support
https://github.com/TheKevJames/coveralls-python/issues/245